migrate_test.go

```diff
@@ -64,13 +64,13 @@ func TestNew(t *testing.T) {
 
 func ExampleNew() {
 	// Read migrations from /home/mattes/migrations and connect to a local postgres database.
-	m, err := New("file:///home/mattes/migrations", "postgres://mattes:secret@localhost:5432/database?sslmode=disable")
+	m, err := migrate.New("file:///home/mattes/migrations", "postgres://mattes:secret@localhost:5432/database?sslmode=disable")
```

Comment:
Tests are already in the `migrate` package, so no need to import it.

Reply:
Thanks, updated. 👍🏼